tiflash/tune-tiflash-performance.md

```diff
@@ -34,6 +34,66 @@ The variable [`tidb_enforce_mpp`](/system-variables.md#tidb_enforce_mpp-new-in-v
 set @@tidb_enforce_mpp = ON;
 ```
 
+The following example shows the query result before and after `tidb_enforce_mpp` is enabled. Before this option is enabled, TiDB needs to read data from TiKV and execute `Join` and `Aggregation` in TiDB. After `tidb_enforce_mpp` is enabled, `Join` and `Aggregation` are pushed down to TiFlash. In addition, because the optimizer does not necessarily generate MPP execution plans, by enabling `tidb_enforce_mpp`, you can force the optimizer to generate `MPP` execution plans.
```

Comment:
```suggestion
The following example shows the query result before and after `tidb_enforce_mpp` is enabled. Before this variable is enabled, TiDB needs to read data from TiKV and execute `Join` and `Aggregation` in TiDB. After `tidb_enforce_mpp` is enabled, `Join` and `Aggregation` are pushed down to TiFlash. In addition, because the optimizer does not necessarily generate MPP execution plans, by enabling `tidb_enforce_mpp`, you can force the optimizer to generate MPP execution plans.
```